### ISSUE TITLE
Post scores to Canvas (Canvas sync)

### ISSUE THREAD

**Issue by atulprak** [eecs-autograder/autograder.io#26]
It would be good if total project scores can be synced/uploaded to Canvas as in Gradescope more easily. That would help immensely in terms of workflow of managing integration of all the individual project score spreadsheets into a common place.

**Comment by james-perretta** [comment]
Agreed, I'll add it to my list of things to take a look at the canvas API
